src/cephadm/cephadm

```diff
@@ -93,7 +92,7 @@ if sys.version_info > (3, 0):
 container_path = ''
 cached_stdin = None
 
-DATEFMT = '%Y-%m-%dT%H:%M:%S.%f'
+DATEFMT = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
Note that we need to continue to parse dates without `Z`

Reply:
Sure, the helper function [str_to_datetime](https://github.com/ceph/ceph/pull/37920/files#diff-601cdd0e9db99fa5fc831bebf2d13d4d85be9d68664ef091ddfee90fcf336beaR26) is able to do that.